Atomic/syscontainers.py

```diff
@@ -669,6 +669,17 @@ def _canonicalize_location(self, destination):
             sysroot.load()
             osname = sysroot.get_booted_deployment().get_osname()
             destination = os.path.realpath(os.path.join("/ostree/deploy/", osname, os.path.relpath(destination, "/")))
+            # Test that the ostree location and /var show the same file.
+            # If they do not then we must return /var.
+            test_file = str(uuid.uuid4())
+            with open(os.path.join(destination, '.' + test_file), 'w') as f:
+                f.write('')
+            if not os.path.isfile('/var/.' + test_file):
+                util.write_out(
+                    '{} does not show the same content as /var. Falling back to /var'.format(destination))
```

Comment:
let's print this only for debugging, as it will be printed each time when a separate `/var` is used.